user/TASK/gimbal_task/gimbal_task.c

```diff
@@ -121,24 +172,63 @@ static void get_new_data(Gimbal_t *gimbal_data){
  */
 static void update_setpoints(Gimbal_t *gimbal_set){
     
-    gimbal_set->yaw_motor.pos_set += (-1) * int16_deadzone(gimbal_set->rc_update->rc.ch[2], -DEADBAND, DEADBAND) / 10;
-    gimbal_set->pitch_motor.pos_set += int16_deadzone(gimbal_set->rc_update->rc.ch[3], -DEADBAND, DEADBAND) / 10;
+    //yaw:
+    //rc -> theta -> complex rotation -> new setpoint
+
+    // TODO: FIX
```

Comment:
Fix what? If minor then should be done before push

Reply:
I'm hoping this is an old fix comment, it did work.....